doc/dev/crimson/crimson.rst

```diff
@@ -16,8 +16,7 @@ Building Crimson
 Crimson is not enabled by default. Enable it at build time by running::
 
   $ WITH_SEASTAR=true ./install-deps.sh
-  $ mkdir build && cd build
```

Comment:
This line is removed completely since the do_make.sh script is in /ceph/ and there's therefore no need to cd into build in order to do the next line.